src/lib/containers/table/SynapseTable.tsx

```diff
@@ -979,36 +988,44 @@ export default class SynapseTable extends React.Component<
       )
     }
     if (dateListColumnIndicies.includes(colIndex)) {
-      const jsonData:number[] = JSON.parse(columnValue)
-      return jsonData.map(
-        (val: number, index: number) => {
-          return <span className={isBold}>
-            {new Date(val).toLocaleString()}{index !== jsonData.length-1 ? ', ' : ''}
+      const jsonData: number[] = JSON.parse(columnValue)
```

Comment:
It seems that we can abstract this into a function that would take column value and type
```
function outputValues<T>(columnValue: string, type: string) {
 return JSON.parse(columnValue).map((val: string, index: number) => {
        return (
          <span className={isBold}>
           // different handling here depending on the type passed  {val ? 'true' : 'false'}
            {index !== jsonData.length - 1 ? ', ' : ''}
          </span>
        )
      })
}```

Reply:
I will punt this to another PR, but that would be a great rewrite